numpy/polynomial/tests/test_chebyshev.py

```diff
@@ -111,6 +113,16 @@ def test_chebdiv(self):
                 res = cheb.chebadd(cheb.chebmul(quo, ci), rem)
                 assert_equal(trim(res), trim(tgt), err_msg=msg)
 
+    def test_chebpow(self):
+        for i in range(5):
+            for j in range(5):
+                msg = "At i=%d, j=%d" % (i, j)
+                c = np.array(range(i + 1))
+                power = j + 1
+                tgt = reduce(cheb.chebmul, [c]*power)
```

Comment:
@eric-wieser, after your comment, I realized i could replace `i` with `_`, since it was not being used, then I wondered why we weren't using reduce, and if it would take a performance hit, so I tried it, and as best as I can tell from lot of `timeit` testing, they are roughly equivlent, though the `reduce` seemed to be performing better as the power grew. Please let me know if we are discouraged from using functools/itertools/collections modules, and I will go back to the for loop and do as you recommended.

Reply:
Using `reduce` here is fine by me.